### ISSUE TITLE
Admin_1_states_provinces: Regions of France was updated in 2016

### ISSUE THREAD

**Issue by KennethSui** [nvkelso/natural-earth-vector#429]
In admin_1_states_provinces data, the region attributes of French provinces are still in the old divisions. Since 2016 France is mainly divided into 18 new administrative regions: 13 regions in metropolitan France (including the territorial collectivity of Corsica).

**Comment by nvkelso** [comment]
Who's On  First has been updated, so pull those updates in here.

**Comment by elliotap** [comment]
Verified

Proposed Changes: Update region names and codes per https://fr.wikipedia.org/wiki/ISO_3166-2:FR, as seen in table below. Region codes will be modified for all features, and region names updated where they were changed ( "-" indicates no change)

For overseas departments, value for region code is currently department code. Proposing to update these also to 3 letter region codes, to standardize length of codes (FR-XYZ).

ne_id | name | type | region | region_cod | New Region | New Region Code
-- | -- | -- | -- | -- | -- | --
1159316815 | Bas-Rhin | Metropolitan département | Alsace | FR-A | Grand Est | FR-GES
1159313579 | Haute-Rhin | Metropolitan département | Alsace | FR-A | Grand Est | FR-GES
1159313549 | Pyrénées-Atlantiques | Metropolitan département | Aquitaine | FR-B | Nouvelle-Aquitaine | FR-NAQ
1159316893 | Landes | Metropolitan département | Aquitaine | FR-B | Nouvelle-Aquitaine | FR-NAQ
1159316895 | Gironde | Metropolitan département | Aquitaine | FR-B | Nouvelle-Aquitaine | FR-NAQ
1159316897 | Dordogne | Metropolitan département | Aquitaine | FR-B | Nouvelle-Aquitaine | FR-NAQ
1159316899 | Lot-et-Garonne | Metropolitan département | Aquitaine | FR-B | Nouvelle-Aquitaine | FR-NAQ
1159313557 | Puy-de-Dôme | Metropolitan département | Auvergne | FR-C | Auvergne-Rhône-Alpes | FR-ARA
1159316881 | Allier | Metropolitan département | Auvergne | FR-C | Auvergne-Rhône-Alpes | FR-ARA
1159316887 | Haute-Loire | Metropolitan département | Auvergne | FR-C | Auvergne-Rhône-Alpes | FR-ARA
1159316885 | Cantal | Metropolitan département | Auvergne | FR-C | Auvergne-Rhône-Alpes | FR-ARA
1159316747 | Manche | Metropolitan département | Basse-Normandie | FR-P | Normandie | FR-NOR
1159313547 | Calvados | Metropolitan département | Basse-Normandie | FR-P | Normandie | FR-NOR
1159316749 | Orne | Metropolitan département | Basse-Normandie | FR-P | Normandie | FR-NOR
1159316823 | Nièvre | Metropolitan département | Bourgogne | FR-D | Bourgogne-Franche-Comté | FR-BFC
1159316827 | Côte-d'Or | Metropolitan département | Bourgogne | FR-D | Bourgogne-Franche-Comté | FR-BFC
1159313559 | Saône-et-Loire | Metropolitan département | Bourgogne | FR-D | Bourgogne-Franche-Comté | FR-BFC
1159316825 | Yonne | Metropolitan département | Bourgogne | FR-D | Bourgogne-Franche-Comté | FR-BFC
1159313259 | Morbihan | Metropolitan département | Bretagne | FR-E | - | FR-BRE
1159316751 | Finistère | Metropolitan département | Bretagne | FR-E | - | FR-BRE
1159316753 | Côtes-d'Armor | Metropolitan département | Bretagne | FR-E | - | FR-BRE
1159316755 | Ille-et-Vilaine | Metropolitan département | Bretagne | FR-E | - | FR-BRE
1159316777 | Indre-et-Loire | Metropolitan département | Centre | FR-F | Centre-Val de Loire | FR-CVL
1159316773 | Indre | Metropolitan département | Centre | FR-F | Centre-Val de Loire | FR-CVL
1159313561 | Loir-et-Cher | Metropolitan département | Centre | FR-F | Centre-Val de Loire | FR-CVL
1159316769 | Loiret | Metropolitan département | Centre | FR-F | Centre-Val de Loire | FR-CVL
1159316771 | Cher | Metropolitan département | Centre | FR-F | Centre-Val de Loire | FR-CVL
1159316767 | Eure-et-Loir | Metropolitan département | Centre | FR-F | Centre-Val de Loire | FR-CVL
1159316801 | Ardennes | Metropolitan département | Champagne-Ardenne | FR-G | Grand Est | FR-GES
1159316805 | Haute-Marne | Metropolitan département | Champagne-Ardenne | FR-G | Grand Est | FR-GES
1159313569 | Marne | Metropolitan département | Champagne-Ardenne | FR-G | Grand Est | FR-GES
1159316803 | Aube | Metropolitan département | Champagne-Ardenne | FR-G | Grand Est | FR-GES
1159308481 | Haute-Corse | Metropolitan département | Corse | FR-H | - | FR-COR
1159317105 | Corse-du-Sud | Metropolitan département | Corse | FR-H | - | FR-COR
1159316817 | Territoire de Belfort | Metropolitan département | Franche-Comté | FR-I | Bourgogne-Franche-Comté | FR-BFC
1159313577 | Doubs | Metropolitan département | Franche-Comté | FR-I | Bourgogne-Franche-Comté | FR-BFC
1159316821 | Jura | Metropolitan département | Franche-Comté | FR-I | Bourgogne-Franche-Comté | FR-BFC
1159316819 | Haute-Saône | Metropolitan département | Franche-Comté | FR-I | Bourgogne-Franche-Comté | FR-BFC
1159315647 | Guadeloupe | Overseas département | Guadeloupe | FR-GP |   | FR-GUA
1159311355 | Guyane française | Overseas département | Guyane française | FR-GF |   | FR-GUF
1159313563 | Eure | Metropolitan département | Haute-Normandie | FR-Q | Normandie | FR-NOR
1159316745 | Seine-Maritime | Metropolitan département | Haute-Normandie | FR-Q | Normandie | FR-NOR
1159316799 | Paris | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159316797 | Hauts-de-Seine | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159316791 | Seine-Saint-Denis | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159316789 | Val-d'Oise | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159316787 | Yvelines | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159316785 | Essonne | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159313565 | Seien-et-Marne | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159316795 | Val-de-Marne | Metropolitan département | Île-de-France | FR-J | - | FR-IDF
1159316863 | Pyrénées-Orientales | Metropolitan département | Languedoc-Roussillon | FR-K | Occitanie | FR-OCC
1159316859 | Gard | Metropolitan département | Languedoc-Roussillon | FR-K | Occitanie | FR-OCC
1159313551 | Hérault | Metropolitan département | Languedoc-Roussillon | FR-K | Occitanie | FR-OCC
1159316861 | Aude | Metropolitan département | Languedoc-Roussillon | FR-K | Occitanie | FR-OCC
1159316857 | Lozère | Metropolitan département | Languedoc-Roussillon | FR-K | Occitanie | FR-OCC
1159313567 | Haute-Vienne | Metropolitan département | Limousin | FR-L | Nouvelle-Aquitaine | FR-NAQ
1159316891 | Corrèze | Metropolitan département | Limousin | FR-L | Nouvelle-Aquitaine | FR-NAQ
1159316889 | Creuse | Metropolitan département | Limousin | FR-L | Nouvelle-Aquitaine | FR-NAQ
1159316807 | Meuse | Metropolitan département | Lorraine | FR-M | Grand Est | FR-GES
1159313581 | Meurthe-et-Moselle | Metropolitan département | Lorraine | FR-M | Grand Est | FR-GES
1159316809 | Moselle | Metropolitan département | Lorraine | FR-M | Grand Est | FR-GES
1159316813 | Vosges | Metropolitan département | Lorraine | FR-M | Grand Est | FR-GES
1159311599 | Martinique | Overseas département | Martinique | FR-MQ | - | FR-MTQ
1159315621 | Mayotte | Overseas département | Mayotte | FR-YT | - | FR-MAY
1159313555 | Haute-Garonne | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316869 | Hautes-Pyrénées | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316867 | Ariège | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316871 | Gers | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316873 | Tarn-et-Garonne | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316879 | Tarn | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316877 | Aveyron | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316875 | Lot | Metropolitan département | Midi-Pyrénées | FR-N | Occitanie | FR-OCC
1159316737 | Nord | Metropolitan département | Nord-Pas-de-Calais | FR-O | Hauts-de-France | FR-HDF
1159313573 | Pas-de-Calais | Metropolitan département | Nord-Pas-de-Calais | FR-O | Hauts-de-France | FR-HDF
1159313545 | Vendée | Metropolitan département | Pays de la Loire | FR-R | Pays de la Loire | FR-PDL
1159316759 | Loire-Atlantique | Metropolitan département | Pays de la Loire | FR-R | Pays de la Loire | FR-PDL
1159316763 | Mayenne | Metropolitan département | Pays de la Loire | FR-R | Pays de la Loire | FR-PDL
1159316765 | Sarthe | Metropolitan département | Pays de la Loire | FR-R | Pays de la Loire | FR-PDL
1159316761 | Maine-et-Loire | Metropolitan département | Pays de la Loire | FR-R | Pays de la Loire | FR-PDL
1159316743 | Aisne | Metropolitan département | Picardie | FR-S | Hauts-de-France | FR-HDF
1159316741 | Somme | Metropolitan département | Picardie | FR-S | Hauts-de-France | FR-HDF
1159313575 | Oise | Metropolitan département | Picardie | FR-S | Hauts-de-France | FR-HDF
1159313543 | Charente-Maritime | Metropolitan département | Poitou-Charentes | FR-T | Nouvelle-Aquitaine | FR-NAQ
1159316783 | Charente | Metropolitan département | Poitou-Charentes | FR-T | Nouvelle-Aquitaine | FR-NAQ
1159316779 | Deux-Sèvres | Metropolitan département | Poitou-Charentes | FR-T | Nouvelle-Aquitaine | FR-NAQ
1159316781 | Vienne | Metropolitan département | Poitou-Charentes | FR-T | Nouvelle-Aquitaine | FR-NAQ
1159316855 | Alpes-Maritimes | Metropolitan département | Provence-Alpes-Côte-d'Azur | FR-U | Provence-Alpes-Côte-d'Azur | FR-PAC
1159316851 | Alpes-de-Haute-Provence | Metropolitan département | Provence-Alpes-Côte-d'Azur | FR-U | Provence-Alpes-Côte-d'Azur | FR-PAC
1159316853 | Hautes-Alpes | Metropolitan département | Provence-Alpes-Côte-d'Azur | FR-U | Provence-Alpes-Côte-d'Azur | FR-PAC
1159313541 | Var | Metropolitan département | Provence-Alpes-Côte-d'Azur | FR-U | Provence-Alpes-Côte-d'Azur | FR-PAC
1159316845 | Bouches-du-Rhône | Metropolitan département | Provence-Alpes-Côte-d'Azur | FR-U | Provence-Alpes-Côte-d'Azur | FR-PAC
1159316849 | Vaucluse | Metropolitan département | Provence-Alpes-Côte-d'Azur | FR-U | Provence-Alpes-Côte-d'Azur | FR-PAC
1159315617 | La Réunion | Overseas département | Réunion | FR-RE | - | FR-LRE
1159316843 | Savoie | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA
1159316841 | Haute-Savoie | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA
1159316839 | Ain | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA
1159310957 | Isère | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA
1159316831 | Drôme | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA
1159316837 | Rhône | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA
1159316833 | Ardèche | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA
1159316835 | Loire | Metropolitan département | Rhône-Alpes | FR-V | Auvergne-Rhône-Alpes | FR-ARA

**Comment by nvkelso** [comment]
I spot checked the changes in this PR and they look great, thanks!

To recap: in 2016 the existing French "departments" (what Natural Earth considers admin-1) were regrouped into new "regions" (which NE tracks as groupings of admin-1), so the work in this PR so far establishes the new region names and region codes in the admin-1 data tables. This took the number of enumerated metropolitan regions from "22 to 13".

One thing to verify together... this PR should be coherent with the boundary lines, too.

_(below) Blue lines are department boundaries, red lines are ALSO region boundaries – but are they pre-2016 regions?_

![image](https://user-images.githubusercontent.com/853051/115346038-7fed6d00-a164-11eb-9b4e-74a038fe6605.png)

Compare with what Wikipedia has for the 13 metropolitan French regions – the NE red lines should match:

![image](https://user-images.githubusercontent.com/853051/115346165-aca18480-a164-11eb-8a9e-fb59e624e7dc.png)

This involves changing the **featurecla** between `Admin-1 boundary` and `Admin-1 region boundary` on the lines in ne_10m_admin_1_states_provinces_lines shapefile. (I think just property updates no geometry updates.)

But it looks like those were changed at some point already? 

Running `git log 10m_cultural/ne_10m_admin_1_states_provinces_lines.dbf` results in this gem:

```
commit d4e97a8bfee2e4bc9f706fb137a65db078979182
Author: Nathaniel Kelso <nvkelso@gmail.com>
Date:   Mon Oct 14 11:19:25 2019 -0700

    fix France region boundaries circa 2016, issue #288
```

Because you're working off the v5-prequel branch you get this change for free (a few months before Arlo was born and I went MIA), even though it hasn't been released yet :)

So I'll merge the PR now!
